x-pack/plugins/task_manager/server/task_store.ts

```diff
@@ -504,6 +505,36 @@ export class TaskStore {
     }
   }
 
+  async msearch(opts: SearchOpts[] = []): Promise<FetchResult> {
```

Comment:
can we add a unit test for this?

Reply:
Just a note this is NOT in 842d4023a447ed91b81627faee668b0bcb2a1e16, as I wanted the functional changes in, I think this test will likely be pretty hairy, and could probably be deferred (but taking a look right now!)

Reply:
Added in https://github.com/elastic/kibana/pull/187809/commits/4b357c1abb8556b91c3b12635136235deab0a7e1.